framework/assets/yii.js

```diff
@@ -472,7 +472,7 @@ window.yii = (function ($) {
                 return true;
             }
 
-            if (message !== undefined) {
+            if (message !== undefined && message !== false && message !== '') {
```

Comment:
How message could be `false`?

Reply:
In case message is set to `false`. See https://api.jquery.com/data/#data2. We're using `data()` to get the value. It converts values automatically.